openmdao/core/tests/test_expl_comp.py

```diff
@@ -980,6 +980,121 @@ def compute_jacvec_product(self, inputs, d_inputs, d_outputs, mode):
         # verify read_only status is reset after AnalysisError
         prob['length'] = 111.
 
+@unittest.skipUnless(MPI, "MPI is required.")
+class TestMPIExplComp(unittest.TestCase):
+    N_PROCS = 4
+
+    def test_list_inputs_outputs_with_parallel_comps(self):
+        class TestComp(om.ExplicitComponent):
+
+            def initialize(self):
+                self.options['distributed'] = False
+
+            def setup(self):
+                self.add_input('x', shape=1)
+                self.add_output('y', shape=1)
+                self.declare_partials('y', 'x')
+
+            def compute(self, inputs, outputs):
+                outputs['y'] = inputs['x'] ** 2
+
+            def compute_partials(self, inputs, J):
+                J['y', 'x'] = 2 * inputs['x']
+
+
+        class TestComp2(om.ExplicitComponent):
+
+            def initialize(self):
+                self.options['distributed'] = False
+
+            def setup(self):
+                self.add_input('x', shape=1)
+                self.add_output('y', shape=1)
+                self.declare_partials('y', 'x')
+
+            def compute(self, inputs, outputs):
+                print(self.comm.rank)
```

Comment:
print left over from debugging?
... but isn't this class identical to TestComp anyway?

Reply:
Yep, fixed